include/seqcmd.h

```diff
@@ -0,0 +1,510 @@
+#ifndef SEQCMD_H
+#define SEQCMD_H
+
+// ==== Primary commands ====
+
+#define SEQCMD_MASK 0xF0000000
```

Comment:
this name may be too generic, `SEQCMD_TYPE_MASK` (like the `SeqCmdType` enum) may be better
(I'm not sure about "type" itself for either, though, "op" like in `SEQCMD_OP_` seems more consistent idk)